src/datasources/lemondb/DDL.sql

```diff
@@ -0,0 +1,89 @@
+
+SET default_tablespace = ts_lemon;
+CREATE EXTENSION IF NOT EXISTS pgcrypto;
+CREATE TABLE IF NOT EXISTS accounts
+(
+    account_id   serial PRIMARY KEY,
+    account_name varchar(255) NOT NULL,
+    login        varchar(100) NOT NULL,
+    password     varchar      NOT NULL
+);
+
+CREATE UNIQUE INDEX IF NOT EXISTS iu_accounts$login ON accounts (login) TABLESPACE ts_lemon_indexes;
+
+ALTER TABLE accounts
+    ADD CONSTRAINT ch_accounts$login
+        CHECK (login ~ '^\S+$');
```

Comment:
What does it mean?

Reply:
this means that there should be no spaces in the login